client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Properties panel quality of life improvements</h4>
-          We removed automatic (magic) sorting from all properties panels. Additional minor improvements ensure that implementing Camunda 7 and 8 is simpler than ever.
+          <h4>Enhanced form component documentation</h4>
+          We've added convenient documentation links directly within the properties panel for Form components. This allows you to quickly find information about the functions and configurations when you need it.
```

Comment:
What does this refer to?

Reply:
Yes, this refers to the form-js feature you linked:
![image](https://github.com/camunda/camunda-modeler/assets/21984219/1419238f-dc19-4881-90db-595861b7eaa9)

Reply:
So basically about the button to the documentation.

Reply:
We could consider to shorten it:

```suggestion
          Documentation links embedded into the properties panel give you quick access to functions and configurations each Form component supports.
```